rosbag2_performance/rosbag2_performance_benchmarking/launch/benchmark_launch.py

```diff
@@ -217,6 +230,10 @@ def _producer_node_exited(event, context):
 
     # Bump up producer index, so the launch sequence can continue
     _producer_idx += 1
+
+    # Give disks some time to flush their internal cache before starting next experiment
+    time.sleep(5)
```

Comment:
Is this really the right way to do this? Is there any way to check it programmatically? I'm always suspicious of arbitrary waits.

Reply:
I don't know of any portable system API to check whether there is something in the disk cache. I am not a fan of arbitrary waits either, but I don't see an alternative atm.

Reply:
Do we have any reason to wait 5 seconds vs any other time? Not going to block this review, but I want to make sure we're at least thinking about hard problems

Reply:
Well, actually that was set based on our intuition. There is surely a place for improvement here, but - what Adam said - that would require some way of inspecting hardware cache.